.github/workflows/release-notes.yaml

```diff
@@ -0,0 +1,13 @@
+name: release-notes
+on: [workflow_dispatch]
+jobs:
+  release-notes:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./pleasew build -p -v2 //cmd/release-notes
+      - run: | 
+          git config --local user.email "action@github.com"
+          git config --local user.name "release-notes-bot"
+          git add RELEASE_NOTES.md
+          git push
```

Comment:
will this trigger the the other workflows with `on: [push]`? Shouldn't be an issue as we _should_ be idempotent 🤞 

How about attaching this to the information to each Release's body (https://github.com/google/go-github/blob/v37.0.0/github/repos_releases.go#L25) so it comes up here https://github.com/thought-machine/falco-probes/releases/tag/85c88952 instead of the default list of commits to master since the last release

Reply:
For the `on: [push]`, it won't trigger other workflows (the token granted is specially configured to avoid this).

For the release body, is that as well as, or instead of? My thinking with the `.md` file in source is that it takes zero API/network requests to check for a probe's existance (so maybe `RELEASED_PROBES.md` is a better name, since they really aren't much like release notes at all). Moving it to the release notes won't add too many more calls, and having it in both places is no problem either, so happy to implement it either way.

Reply:
nice 👌 

Ooh, I was originally thinking just the release body, but I like the `RELEASED_PROBES.md` idea too. Let's start with the `RELEASED_PROBES.md` way and we can always add the information elsewhere in the future :)

Reply:
'Just write an .md file to the repo and read from disk' gets pretty ugly in the world of Please. Since it actually only takes 1 API req to get all the release notes from all the github releases, it def wasn't worth the effort. Will just write to the body as you originally recommended in the issue 🙃